modules/architecture-machine-roles.adoc

```diff
@@ -58,6 +59,7 @@ Compute machine sets are groupings of compute machine resources under the `machi
 == Cluster masters
 
 In a Kubernetes cluster, the control plane nodes run services that are required to control the Kubernetes cluster. In {product-title}, the control plane machines are the control plane. They contain more than just the Kubernetes services for managing the {product-title} cluster. Because all of the machines with the control plane role are control plane machines, the terms _master_ and _control plane_ are used interchangeably to describe them. Instead of being grouped into a compute machine set, control plane machines are defined by a series of standalone machine API resources. Extra controls apply to control plane machines to prevent you from deleting all control plane machines and breaking your cluster.
+//todo: assess for CPMS impact. 4.11: https://docs.openshift.com/container-platform/4.11/architecture/control-plane.html#defining-masters_control-plane
```

Comment:
Architecture > Control plane architecture > [Machine roles in OpenShift Container Platform](https://53397--docspreview.netlify.app/openshift-enterprise/latest/architecture/control-plane.html#architecture-machine-roles_control-plane)

Reply:
This also seems to have some muddled terms issues around "master" (k8s term), "control plane", and "`master` role". I think the thing to do here is to add info about control plane machine sets and see if the paragraph can be cleaned up a bit wrt terminology.